examples/ExampleLoginPlugin.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable */
+import { Component } from 'react';
+
+class ExampleLoginPlugin extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    const { onPermissionOverrides } = this.props;
+    if (onPermissionOverrides) {
+      const permissionOverrides = {
+        canUsePanels: true,
+        canCreateDashboard: true,
+        canCreateCodeStudio: true,
+        canCreateQueryMonitor: true,
+        canCopy: true,
+        canDownloadCsv: true,
+      };
+      onPermissionOverrides((username, operateAs, groups) => {
+        console.log('[ExampleLoginPlugin]', username, operateAs, groups);
+        return permissionOverrides;
+      });
+    }
```

Comment:
Do we have to check if the prop is non-null here? All Login plugins should get this prop. I think this example would be a lot simpler if it was just:
```suggestion
    const { onPermissionOverrides } = this.props;
    onPermissionOverrides((username, operateAs, groups) => {
      console.log('[ExampleLoginPlugin]', username, operateAs, groups);
      return {
        canUsePanels: true,
        canCreateDashboard: true,
        canCreateCodeStudio: true,
        canCreateQueryMonitor: true,
        canCopy: true,
        canDownloadCsv: true,
      };
    });
```

Reply:
Fixed